src/lib/ngsi/EntityId.cpp

```diff
@@ -121,7 +119,7 @@ std::string EntityId::render
        indent2 += "  ";
     }
 
-    out += (isInVector? indent + (isAssoc? "\"" + assocTag + "\" : ": "") + "{\n": "");
+    out += (isInVector? indent + "" + "{\n": "");
```

Comment:
indent + "" ? Probably not necessary ... :-)
Also, please use a space before and after ':' in these 'special ifs'.